packages/block-renderer/src/components/block-renderer-container.tsx

```diff
@@ -88,19 +88,22 @@ const ScaledBlockRendererContainer = ( {
 	}, [] );
 
 	const scale = containerWidth / viewportWidth;
+	const maxHeightFor100vh = Math.max(
+		contentHeight || 0,
```

Comment:
It's called `maxHeightFor100vh` because but could be just `heightFor100vh`. 
The logic is that the greatest height should be used. 
They can be `null` or `undefined`, that's why I need `|| 0`.